app/js/miniQuery.js

```diff
@@ -0,0 +1,121 @@
+//OLD JS
+// Release 0: A selector library
+
+var miniQuery = (function(){
+
+  var exports = {}
+
+  exports.select = function(element){
+    if(element.charAt(0) === "#"){
+      var idsArray = []
+      idsArray.push(document.getElementById(element.substring(1)))
+      return idsArray
+    } else if(element.charAt(0) === "."){
+      return document.getElementsByClassName(element.substring(1))
+    } else{
+      return document.getElementsByTagName(element)
+    }
+  }
+
+  exports.hide = function(element){
+    var item = this.select(element);
+    for(var i = 0; i < item.length; i++){
+      item[i].style.display = "none";
+    }
+  }
+
+  exports.show = function(element){
+    var item = this.select(element);
+    for(var i = 0; i < item.length; i++){
+      item[i].style.display = "block";
```

Comment:
what if the style was `inline` previously? take a peek at how jquery handles it: https://github.com/jquery/jquery/blob/5904468b9c49d7d5a780010f44d7e76dd4c81706/src/css.js#L68

they define a special `data-` attribute on the tag with the old display value